docs/users_guide/ocean/test_groups/baroclinic_gyre.rst

```diff
@@ -0,0 +1,180 @@
+.. _baroclinic_gyre:
+
+baroclinic_gyre
+===============
+
+The ``baroclinic_gyre`` test group implements variants of the
+baroclinic ocean gyre set-up from the 
+`MITgcm test case`_.
+
+This test simulates a baroclinic, wind and buoyancy-forced, double-gyre ocean circulation. 
+The grid employs spherical polar coordinates with 15 vertical layers.
+The configuration is similar to the double-gyre setup first solved numerically
+in `Cox and Bryan (1984) <https://journals.ametsoc.org/view/journals/phoc/14/4/1520-0485_1984_014_0674_anmotv_2_0_co_2.xml>`_: the model is configured to
+represent an enclosed sector of fluid on a sphere, spanning the tropics to mid-latitudes,
+:math:`60^{\circ} \times 60^{\circ}` in lateral extent.
+The fluid is :math:`1.8`\ km deep and is forced by a zonal wind
+stress which is constant in time, :math:`\tau_{\lambda}`, varying sinusoidally in the
+north-south direction.
+
+.. figure:: ./images/baroclinic_gyre_config.png
```

Comment:
We typically limit images to 500 pixels along the longer size to keep the repo small.  This file isn't super large but what do you think about shrinking it by a factor of 4?